src/app/mytools/mytools.service.ts

```diff
@@ -14,17 +14,46 @@
  *    limitations under the License.
  */
 
+import { HttpErrorResponse } from '@angular/common/http';
 import { Injectable } from '@angular/core';
+import { AlertService } from 'app/shared/alert/state/alert.service';
+import { ContainerService } from 'app/shared/container.service';
 import { EntryType } from 'app/shared/enum/entry-type';
+import { MyEntriesStateService } from 'app/shared/state/my-entries.service';
+import { UsersService } from 'app/shared/swagger';
+import { finalize } from 'rxjs/operators';
 import { MyEntriesService } from './../shared/myentries.service';
 
 @Injectable()
 export class MytoolsService extends MyEntriesService {
+  constructor(
+    private alertService: AlertService,
+    private usersService: UsersService,
+    private containerService: ContainerService,
+    private myEntriesService: MyEntriesStateService
+  ) {
+    super();
+  }
   getGroupIndex(groupEntries: any[], group: string): number {
     return groupEntries.findIndex(nsContainer => nsContainer.namespace === group);
   }
 
-  getMyEntries(userId: number, entryType: EntryType) {}
+  getMyEntries(userId: number, entryType: EntryType) {
```

Comment:
I wouldn't really rely on any of this to determine whether it's fetching or not.  Seems rather dangerous and it may be fetching a different set of entries for all we know

Reply:
Ah right, you'd have to specify the type when calling setRefreshingMyEntries.